support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -102,3 +108,49 @@ test.describe('Subscribe/Contribute via the Tiered checkout)', () => {
 		});
 	});
 });
+
+test.describe('Subscribe (S+) incl PromoCode via the Tiered checkout', () => {
+	testDetailsPromo.forEach((testDetails) => {
+		test(`${testDetails.frequency} (S+) Subscription incl PromoCode at Tier-2 with Credit/Debit card - UK`, async ({
```

Comment:
I'm trying to work out what the user-centric description of this is - is it just that they see the promo description text?

Reply:
Definitely - let's look at this after.